lib/matplotlib/backends/backend_macosx.py

```diff
@@ -104,10 +106,15 @@ def remove_rubberband(self):
         self.canvas.remove_rubberband()
 
     def save_figure(self, *args):
+        directory = os.path.expanduser(mpl.rcParams['savefig.directory'])
```

Comment:
Is it possible to add a smoke test for this in the tests?

Reply:
Unfortunately I think this is another PR that would be quite hard to add a test for because it is using the GUI :( I did a global search and didn't see any of the other backends testing this either...

Reply:
I take that back, I just pushed up a test to hijack the GUI function and return a well-defined string instead. I _think_ that should get codecov for the Python branches at least... Still not testing the Objective-C updates, but better than nothing?